src/part-guide/more-async-await.md

```diff
@@ -115,7 +115,7 @@ loop {
 }
 ```
 
-To implement `break` you would need to test the value of the block (a common idiom is to use [`ControlFlow`](https://doc.rust-lang.org/std/ops/enum.ControlFlow.html) for the value of the block, which also allows use of `?`).
+To implement `break` you would need to test the value of the block (a common idiom is to use [`ControlFlow`](https://doc.rust-lang.org/std/ops/enum.ControlFlow.html) for the return value of the block, which also allows use of `?`).
```

Comment:
I think this is not a correct change since the block assumes the value it doesn't return it.